src/gitProjectManager.ts

```diff
@@ -284,7 +284,7 @@ export default class GitProjectManager {
 
     };
 
-    openRecentProjects() {
+    openRecentProjects(openInNewWindow: boolean = false) {
```

Comment:
I changed to add openInNewWindow boolean argument to reflect user setting to this function.